### PR TITLE
[TWEAK] Makes Laces less painful to use.

### PR DESCRIPTION
**Pull request by ShadeAware**
## About The Pull Request
Laces are now smaller and more sturdy against damage (**SIZE** is **DOWN** from 95 to 30. **MAX_DAMAGE** is **UP** from 30 to 50).

Losing your lace no longer instantly kills you; instead, it damages your brain by 10 and gives you a seizure.

Losing your lace now has a new unique sound. 
## Why It's Good For The Game
Making something that is ostensibly supposed to make dying less likely, MORE likely to kill you, is a bit fucking shit. This makes laces worth actually having again and not an actual detriment to you, the player.
## Did You Test It?
No, but its like... a proc change and some value tweaks.
## Authorship
ShadeAware
## Changelog

:cl:
tweak: Laces are less painful to use. Smaller (less likely to be hit), more sturdy, and now don't kill you when they break. Not directly, anyway.
/:cl:

### CODE REVIEW

**Comment by Pariah919** [comment]
```
Losing your lace no longer instantly kills you; instead, it damages your brain by 10 and gives you a seizure.

Losing your lace now has a new unique sound.
```
Losing what is effectively your conciousness should absolutely kill you in my opinion, my problems is that lace kills are definitely too common under current stats, I'd even argue it should be common to hit, but difficult to actually fully damage to a point of where it explodes.

My personal takes, as a medbay player and a command player who mostly uses laces since there's no real reason not to.

People actually dying is incredibly difficult due to how brainmed works, unless you're swamped with 4+ patients and have to let people die, most of the crew has laces for good reason. I don't see why laces should be this effective as they are, death is a part of the game. Losing is expected (and in my opinion usually fun. But that's 100% subjective.)

Keep laces as an instant kill, its too easy to use them as a crutch in combat, up their max health and make them harder to hit. Give them other downsides like EMPs, Skill point costs.

Make losing your lace not break the lace at all, but instead do massive structual damage and really fuck you up once you hit that threshold, and add red grab moves like Jeff suggested for actual executions that reveal/destory/rip out the lace for taking out.

**Comment by ShadeAware** [comment]
> ```
> Losing your lace no longer instantly kills you; instead, it damages your brain by 10 and gives you a seizure.
> 
> Losing your lace now has a new unique sound.
> ```
> 
> Losing what is effectively your conciousness should absolutely kill you in my opinion, my problems is that lace kills are definitely too common under current stats, I'd even argue it should be common to hit, but difficult to actually fully damage to a point of where it explodes.
> 
> My personal takes, as a medbay player and a command player who mostly uses laces since there's no real reason not to.
> 
> Keep laces as an instant kill, its too easy to use them as a crutch in combat, up their max health and make them harder to hit. Give them other downsides like EMPs, Skill point costs.
> 
> Make losing your lace not break the lace at all, but instead do massive structual damage and really fuck you up once you hit that threshold, and add red grab moves like Jeff suggested for actual executions that reveal/destory/rip out the lace for taking out.

It is still entirely possible to destroy a Lace, and easy to do so if you have someone on the ground. At current levels, killing someone is as easy as getting a lucky headshot with a pistol as you are **nigh certain** to hit and their damage threshold is essentially glass. 

Neural Laces are supposed to offer a way to prevent death. Making it easier to kill someone with it defeats the entire point, and at that rate, you may as well remove them.

**Comment by Sparrowbird611** [comment]
Seems kind of early for this. I agree with what pariah said about them killing you. I'd recommend a more moderate buff, 40 HP and 50 size. Still substantially smaller than a brain and a tad more durable, requiring at least three shots from most firearms to kill you. Lace will still make you substantially more resistant to dying from accidental causes, but if someone is actively trying to kill you and target your lace, it's fine for it to make it easier.

**Comment by ShadeAware** [comment]
> Seems kind of early for this. I agree with what pariah said about them killing you. I'd recommend a more moderate buff, 40 HP and 50 size. Still substantially smaller than a brain and a tad more durable, requiring at least three shots from most firearms to kill you.

Laces immediately killing you is right out. I am not going to discuss that.

**Comment by Pariah919** [comment]
>It is still entirely possible to destroy a Lace, and easy to do so if you have someone on the ground. At current levels, killing someone is as easy as getting a lucky headshot with a pistol as you are nigh certain to hit and their damage threshold is essentially glass.

>Neural Laces are supposed to offer a way to prevent death. Making it easier to kill someone with it defeats the entire point, and at that rate, you may as well remove them.

Disagree. Laces are there to prevent death from idiotic mistakes, accidents or plain O2 loss. They're not there to stop you from dying in a gunfight or other such things. And players should be forced to make choices whether to use them or not for the disadvantage.

I don't mind the idea of the original Lace changes, and while I think the statistics are honestly pretty bad on paper and reality. I have to say that losing a lace should be incredibly devastating, and be a incredibly terrible experience for the player, such as death. Even if I disagree with the way it was gone about.

losing laces should be restricted to getting your head blown off and it being stolen, dying from an insane amount of gunfire to the head and or getting coup de grace'd as jeff suggested. But they should be damageable, EMPable and there *should* be downsides to having one.

**Comment by Teragenic** [comment]
I think that laces should atleast be a paralyze if you do not wish to keep them as a instant kill. They are still effectively, your consciousness, I agree with buffing them and making them a bit smaller. Losing a lace shouldn't just be brain damage and a seizure, how would this work for cloned people in the IC sense?

Did the lace already imprint with the cloned person? Mainly, buff the downside of losing the lace. I agree with buffing the health and such of the lace.

**Comment by Sparrowbird611** [comment]
Very early to make this change, regardless of any opinions on it. Change the hitrate of the lace since that was an accidental test value (should be slightly more meaningful IMO, 40 would be good), and buff it's HP in a lower level if you want (+10 HP is the most that should be done) and leave the instant death, this change hasn't even been out for a day yet.

Being able to be resurrected from anything that doesn't destroy your lace is powerful, it's fine to be substantially die when it breaks in exchange for the advantages it gives.

**Comment by ShadeAware** [comment]
> I think that laces should atleast be a paralyze if you do not wish to keep them as a instant kill. They are still effectively, your consciousness, I agree with buffing them and making them a bit smaller. Losing a lace shouldn't just be brain damage and a seizure, how would this work for cloned people in the IC sense?
> 
> Did the lace already imprint with the cloned person? Mainly, buff the downside of losing the lace. I agree with buffing the health and such of the lace.

Laces give you a seizure, which paralyzes the player amongst other things.

**Comment by Rayfod** [comment]
I will not mind if this change is added. Suppose someone is going to kill the person.
Then they will have plenty of time to do it while they are stunned for the 14~ ish seconds it takes for a seizure to stop.
Also, ten brain damage is a little on the low side if we talk about brain damage.
25 brain damage will do better for the brain damage.

**Comment by Yurek1** [comment]
As much as I admit to being a bit of a lacebaby, only giving 10 brain and a seizure? Nah. Leave it an instant kill, but make it hard to do. I agree with this PR otherwise.

**Comment by anconfuzedrock** [comment]
note that the lace has 50% damage reduction, so this effectively brings its health from 60 to 100.

**Comment by Yurek1** [comment]
> note that the lace has 50% damage reduction, so this effectively brings its health from 60 to 100.

honestly i see that as fine, since currently you can blow someone's lace out with a mk59 and a single bullet- it should be something you need to put a little effort into (not too much though)

File: code/modules/organs/internal/stack.dm [hunk 1]
```diff
@@ -63,12 +63,15 @@
...
+	playsound(src,'sound/effects/lace_destroyed.ogg', 70, 0)
 	gore = new /obj/item/weapon/material/shard(get_turf(owner), MATERIAL_GLASS)
 	gore.throw_at(get_edge_target_turf(src,pick(GLOB.alldirs)),rand(1,3),30)
 	gore = new /obj/effect/decal/cleanable/blood/gibs(get_turf(owner))
 	gore.throw_at(get_edge_target_turf(src,pick(GLOB.alldirs)),rand(1,3),30)
 	owner.visible_message(SPAN_WARNING("[owner]'s neck explodes in a shower of strange blue liquid and metallic fragments!"))
-	owner.death()
+	owner.emote("scream")
+	owner.seizure() //Would also include owner.Paralyze but seizure does that anyway.
+	owner.adjustBrainLoss(10)
```
Comment (ProbablyCarl) [inline code/modules/organs/internal/stack.dm:74]: Pump this way the heck up. Please.
It's low enough that you can straight up ignore it, with some minor inconveniences.

**Comment by ProbablyCarl** [comment]
In line with #1624, I will be closing this and put up my own edits. Good stuff otherwise, genuinely.

**Pull request closed without merging**